### DISCUSSION TITLE
Should we provide `EpiNow2` on conda forge?

### DISCUSSION THREAD

**Discussion by jamesmbaazam** [epiforecasts/EpiNow2#677] [Installation]
A user recently tried to install the package on [conda forge](https://conda-forge.org/docs/) based on the LSHTM HPC instructions provided [here](https://hpcinfo.lshtm.ac.uk/applications/rcluster.html#installing-r-packages-via-conda) but that failed because the package isn't provided there. Do you think we should consider doing this?

The conda forge community has done a great job providing templates and a guide for contributing a package. See [here](https://github.com/bgruening/conda_r_skeleton_helper/blob/main/run.R) for the R template and [here](https://conda-forge.org/docs/maintainer/adding_pkgs/) for the contribution guide.

Are there reasons why this is not a good idea?

**Comment by seabbs** [comment]
Aside from more maintenance burden not that I can think of.

**Reply by jamesmbaazam** [reply]
Yes, that's definitely an important consideration.

**Comment by sbfnk** [comment]
Don't know much about conda forge but that looks like it might be a good idea if it helps people install - just to check, though: you could just install via `install.packages()`, right? Is there any particular advantage to using conda forge?

**Reply by jamesmbaazam** [reply]
I'm not sure but I think the conda forge route is useful if there are admin rights issues with installations?

**Reply by seabbs** [reply]
its useful if people are using conda

**Comment by jamesmbaazam** [comment]
I don't think that there's a strong need for it, so maybe we can close this discussion and revisit if it becomes a need.
